src/Cmixin/Holidays/de-national.php

```diff
@@ -2,31 +2,14 @@
 
 return [
     'new-year'                => '01-01',
-    '02-14'                   => '02-14',
-    'easter-52'               => '= easter -52',
-    'easter-48'               => '= easter -48',
-    'easter-46'               => '= easter -46',
-    'easter-3'                => '= easter -3',
     'easter-2'                => '= easter -2',
     'easter'                  => '= easter',
     'easter-p1'               => '= easter 1',
     '05-01'                   => '05-01',
```

Comment:
Sounds like _Tag des Grundgesetzes_ is missing here.
```suggestion
    '05-01'                   => '05-01',
    '05-23'                   => '05-23',
```